arex-instrumentation-api/src/main/java/io/arex/inst/runtime/config/Config.java

```diff
@@ -34,6 +37,7 @@ public static Config get() {
         this.enableDebug = enableDebug;
         this.serviceName = serviceName;
         this.entities = entities;
+        this.genericReturnTypeEntities = entities == null ? Collections.emptyList() : entities.stream().filter(DynamicClassEntity::isGenericReturnType).collect(Collectors.toList());
```

Comment:
This filtering logic can be placed in `getGenericReturnTypeEntities()`, not need to define the `genericReturnTypeEntities` variable and reduces the complexity of the constructor.